doc/CommonTraps.md

```diff
@@ -48,3 +48,9 @@ This document details the common pitfalls when developing for PowerPointLabs, so
 	Step 3. Navigate to Common Properties > Startup Project.
 	
 	Step 4. Select Single startup project and choose PowerPointLabs from the drop down list.
+
+1. To group multiple `Shape` objects together when you only have the references to the objects, an easy way is to extract a `ShapeRange` object from the slide based on the names of the shapes, and then call the `Group()` method on that `ShapeRange`
```

Comment:
There seems to be 2 main types of points: Tips when implementing functionality, and bugs. The tips seems to mostly start with `When`. Let me know what you think.

Reply:
I think you make a good point about the need to be consistent with the words used. Fixed.

We should also keep this in mind if we do intend to restructure the documents!